Framework/Geometry/src/Instrument/InstrumentDefinitionParser.cpp

```diff
@@ -2272,18 +2288,50 @@ void InstrumentDefinitionParser::setLogfile(
       continue;
     }
 
-    // if more than one <value> specified for a parameter use only the first
-    // <value> element
+    DateAndTime currentValidFrom;
+    DateAndTime currentValidTo;
+    currentValidFrom.setToMinimum();
+    currentValidTo.setToMaximum();
+
+    // if more than one <value> specified for a parameter, check the validity
+    // range
     if (numberValueEle >= 1) {
```

Comment:
Should this not be strict inequality? I see it was >= before.

Reply:
No, it should stay >=. Before, if there was one or more values, the first one was picked. Now, if there is one or more values, we check for validity.